src/components/Contact/ContactForm.tsx

```diff
@@ -0,0 +1,101 @@
+'use client'
+
+import ValidatedInput from '../Form/ValidatedInput'
+import { validateEmail } from '../Form/Validators'
+import { useEffect, useState } from 'react'
+import ValidatedTextarea from '@/components/Form/ValidatedTextarea'
+import SubmitButton from '../Form/SubmitButton'
+import { Octokit } from 'octokit'
+
+export default function ContactForm() {
+    const [email, setEmail] = useState('')
+    const [subject, setSubject] = useState('')
+    const [message, setMessage] = useState('')
+    const [formValid, setValid] = useState(false)
+
+    useEffect(() => {
+        if (
+            validateEmail(email) &&
+            subject.trim().length > 0 &&
+            message.trim().length > 0
+        ) {
+            setValid(true)
+        } else {
+            setValid(false)
+        }
+    }, [email, subject, message])
+
+    const createIssue = async () => {
+        const octokit = new Octokit({
+            auth: import.meta.env.VITE_GH_TOKEN,
+        })
+
+        const data = await octokit.request(
+            'POST /repos/help-is-here/hih/issues',
+            {
+                owner: 'help-is-here',
+                repo: 'hih',
+                title: subject,
+                body: `from:${email} message: ${message}`,
+                labels: ['info request'],
+                headers: {
+                    'X-GitHub-Api-Version': '2022-11-28',
+                },
+            }
+        )
+        const addQuery = `
+                mutation {
+                    addProjectV2ItemById(
+                        input: {
+                            projectId: "PVT_kwDOCTO1FM4AZ1WC"
```

Comment:
Does this need to be an env var?

Reply:
It can be but it is not really related to the env the app is running in. I can add it to a constants file since it's a constant if we want.